configmap/hash-gen/main.go

```diff
@@ -74,7 +76,7 @@ func process(data []byte) ([]byte, error) {
 		return nil, errors.New("'metadata.labels' not found")
 	}
 
-	checksum := configmap.Checksum(example.Value)
+	checksum := configmap.Checksum(strings.TrimRightFunc(example.Value, unicode.IsSpace))
```

Comment:
Should we trim the value in the checksum generator itself and trim it left and right for good measure? Seems like no harm done if whitespace is added and I wonder if that could accidentally happen during edits :thinking:

Reply:
Sure sounds good.